src/aiperf/accuracy/graders/lighteval_grader.py

```
@@ -0,0 +1,275 @@
+        """Run lighteval's metric.compute with crash-safety.
+
+        lighteval can raise on malformed predictions (e.g. when its
+        sympy-backed extractor times out or hits an unparseable LaTeX
+        construct). We catch and report ``unparsed=True`` rather than
+        crashing the record processor.
+        """
+        try:
+            doc = self._build_doc(ground_truth)
+            response = ModelResponse(text=[response_text])
+            return float(self._metric.compute(doc, response))
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)